### DISCUSSION TITLE
Perfect French translation - not an issue

### DISCUSSION THREAD

**Discussion by djdakta** [olivierlacan/keep-a-changelog#514] [General]
Hi,

This is not an issue/bug but just a small message to say that the french translation is Perfect!
I'm a native french speaker and (not to brag or anything) also master in french grammar and spelling.
It's very impressive! :-)
Chapeau à/aux traducteur(s)!

**Comment by olivierlacan** [comment]
Well, thanks. That's because je suis français aussi en fait 😉 

![](https://hugelolcdn.com/i/532921.jpg)

**Comment by djdakta** [comment]
Héhé, that explique beaucoup! :-)
